tests/Service/SnippetGeneratorTest.php

```diff
@@ -74,117 +181,185 @@ public function testGenerateAllSnippetsWithFeaturedBadges(): void
+                    ['latest' => 'stable', 'repository' => 'vendor/package'],
+                    RouterInterface::ABSOLUTE_URL,
+                ],
+                [
+                    'pugx_badge_download',
+                    ['repository' => 'vendor/package'],
+                    RouterInterface::ABSOLUTE_URL,
+                ],
+                [
+                    'pugx_badge_version_latest',
+                    ['latest' => 'unstable', 'repository' => 'vendor/package'],
+                    RouterInterface::ABSOLUTE_URL,
+                ],
+                [
+                    'pugx_badge_license',
+                    ['repository' => 'vendor/package'],
+                    RouterInterface::ABSOLUTE_URL,
+                ],
+                [
+                    'pugx_badge_download_type',
+                    ['repository' => 'vendor/package', 'type' => 'monthly'],
+                    RouterInterface::ABSOLUTE_URL,
+                ],
+                [
+                    'pugx_badge_download_type',
+                    ['repository' => 'vendor/package', 'type' => 'daily'],
+                    RouterInterface::ABSOLUTE_URL,
+                ],
+                [
+                    'pugx_badge_version',
+                    ['repository' => 'vendor/package'],
+                    RouterInterface::ABSOLUTE_URL,
+                ],
+                [
+                    'pugx_badge_require',
+                    ['repository' => 'vendor/package', 'type' => 'php'],
                     RouterInterface::ABSOLUTE_URL,
                 ],
                 [
-                    'badge_2_route',
-                    ['routeParam1' => 'param1Value'],
+                    'pugx_badge_composerlock',
+                    ['repository' => 'vendor/package'],
                     RouterInterface::ABSOLUTE_URL,
                 ],
+                [
+                    'pugx_badge_gitattributes',
+                    ['repository' => 'vendor/package'],
+                    RouterInterface::ABSOLUTE_URL,
+                ],
+                [
+                    'pugx_badge_dependents',
+                    ['repository' => 'vendor/package'],
+                    RouterInterface::ABSOLUTE_URL,
+                ],
+                [
+                    'pugx_badge_suggesters',
+                    ['repository' => 'vendor/package'],
+                    RouterInterface::ABSOLUTE_URL,
+                ],
+                [
+                    'pugx_badge_circleci',
+                    ['repository' => 'vendor/package'],
+                    RouterInterface::ABSOLUTE_URL,
+                ]
             )
-            ->willReturnOnConsecutiveCalls('repo_url', 'img_url', 'img_url2');
+            ->willReturnOnConsecutiveCalls('repo_url', 'img_url', 'img_url2', 'img_url', 'img_url3', 'img_url4', 'img_url5', 'img_url6', 'img_url7', 'img_url8', 'img_url9', 'img_url10', 'img_url11', 'img_url12');
```

Comment:
`img_url` two times?

Reply:
I fixed the `img_url` string, I used the same because it's the same badge but, it's better to differentiate all images like your suggestion @JellyBellyDev .